Maple2Storage/Types/Coord.cs

```diff
@@ -218,6 +221,16 @@ public static CoordS From(int x, int y, int z)
         };
     }
 
+    public static CoordS From(float x, float y, float z)
+    {
+        return new()
+        {
+            X = (short) x,
+            Y = (short) y,
+            Z = (short) z
```

Comment:
you might end up with off-by-one values doing this because float math is bad and will give you numbers like "1.9999999999" -> 1